src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/PackageMaps/WinFormsPackageMap.json

```diff
@@ -0,0 +1,24 @@
+﻿[
+  {
+    "PackageSetName": "WinForms",
+    "NetCorePackagesWorkOnNetFx": false,
+    "PackageSetWarning": "System.Windows.Forms.DataVisualization is deprecated, for more details - https://github.com/dotnet/winforms-datavisualization",
```

Comment:
This name seems weird. Why not just `Warning`?

Also, is there a reason why it's automatically a warning? Why not an object that provides the ability to set the notification level (or build break risk)? ie

```
"Message": "...."
"Risk": "...."
```

Reply:
Makes sense, changed the structure to have a message and risk.